### PR TITLE
Point changes

### PR DESCRIPTION
**Pull request by isaac-tfn**
Describe Changes
---

Task Checklist
----
- [x] Have unittests been added (testing individual functions and their edge cases)
- [x] Have integration tests been added (if applicable, to test modules of functionality)
- [ ] Updated RELEASE.md to reflect changes in PR
- [x] Have new dependencies been added?
  - [x] Have they been added to either `requirements.txt` or `requirements_dev.txt`.
  - [x] Have they been added to `setup.cfg`

### CODE REVIEW

File: src/caf/space/weighted_funcs.py [hunk 1]
```diff
@@ -4,21 +4,23 @@
...
 
 # pylint: enable=import-error
 
-from caf.space import inputs, zone_correspondence
+from caf.space import inputs
 
 ##### CONSTANTS #####
 logging.captureWarnings(True)
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger("SPACE")
```
Comment (BenTaylor-TfN) [inline src/caf/space/weighted_funcs.py:23]: How do you expect this logger to work? I got the impression it was inherently part of the ZoneTranslation class, so would expect it to be in there?

Comment (isaac-tfn) [reply src/caf/space/weighted_funcs.py:23]: I was thinking I needed to create the logger in each file for it to be found when called in ZoneTranslation. Think I'm still not completely sure how logging works and got it to a point it worked then didn't touch it again!

Comment (BenTaylor-TfN) [reply src/caf/space/weighted_funcs.py:23]: I've seen this pop up a few times, all with the same name. I think the module loggers need to go back to `__name__`, while the ZoneTranslation class is set to start / end a logger for the function calls it is set to log information for.

**Review by BenTaylor-TfN — APPROVED** [review]
I've ignored the UI stuff, as I don't know anything about that.

Perhaps there's an item for the backlog here to improve the overall test coverage of this project, it's only at 56% currently.

I think the logging needs thinking about in a bit more detail, but happy for that to become an issue to be picked up later
